rasa/utils/tensorflow/crf.py

```diff
@@ -13,8 +13,10 @@
 class CrfDecodeForwardRnnCell(tf.keras.layers.AbstractRNNCell):
     """Computes the forward decoding in a linear-chain CRF."""
 
+    # TODO: the way transition params are initialised in Windows doesn't pass the type check for TensorLike.
+    # I'll look into it later.
     @typechecked
-    def __init__(self, transition_params: TensorLike, **kwargs) -> None:
+    def __init__(self, transition_params, **kwargs) -> None:
```

Comment:
@tabergma could you please take a look at that?

Reply:
Do we get any error message for this? Are all requirements up-to-date? 
`TensorLike` covers the following types
```
TensorLike = Union[
    List[Union[Number, list]],
    tuple,
    Number,
    np.ndarray,
    tf.Tensor,
    tf.SparseTensor,
    tf.Variable,
]
```
In `crf.py` I guess all parameters defined as `TensorLike` could also be directly set to `tf.Tensor` if that would help.

Reply:
I think my laptop became friendlier to me after the holidays and everything started working... 🤷‍♀️